src/field_theory/finite/polynomial.lean

```diff
@@ -188,9 +202,13 @@ end
 
 lemma ker_evalₗ : (evalᵢ σ K).ker = ⊥ :=
 begin
-  refine injective_of_surjective _ _ _ (range_evalᵢ _ _),
-  { rw [dim_R], exact cardinal.nat_lt_omega _ },
-  { rw [dim_R, dim_fun, dim_of_field, mul_one] }
+  rw ker_eq_bot_iff_range_eq_top_of_findim_eq_findim _,
+  refine range_evalᵢ _ _,
+  { rw [finite_dimensional.finite_dimensional_iff_dim_lt_omega, dim_R],
+    exact cardinal.nat_lt_omega _ },
+  { rw [finite_dimensional.finite_dimensional_iff_dim_lt_omega, dim_fun, dim_of_field, mul_one],
+    exact cardinal.nat_lt_omega _ },
+  { rw [finite_dimensional.findim_fintype_fun_eq_card, findim_R] }
```

Comment:
If we add the above instance, this can be golfed to:
```suggestion
  refine (ker_eq_bot_iff_range_eq_top_of_findim_eq_findim _).mpr (range_evalᵢ _ _),
  rw [finite_dimensional.findim_fintype_fun_eq_card, findim_R]
```